documentation/contributing/code/index.markdown

```diff
@@ -150,3 +150,26 @@ You can make a specific clang-tidy build with:
 catkin config --cmake-args -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_CXX_CLANG_TIDY=clang-tidy
 catkin build
 ```
+
+### Exceptions to clang-tidy
+
+In some cases, the warnings generated by clang-tidy may be false-positive, e.g. `performance-unnecessary-copy-initialization` may falsely convert a `non-const` variable to `const`, or cannot be silenced for some reason.
+
+It is recommended to suppress undesired clang-tidy checks by using `NOLINT` or `NOLINTNEXTLINE` comments followed by the list of check names in parentheses:
```

Comment:
```suggestion
It is possible to suppress false-positive clang-tidy checks by using `NOLINT` or `NOLINTNEXTLINE` comments followed by the list of check names in parentheses:
```

Reply:
Here I use `undesired` because some cases may not be false-positive but not desired for some reason. By using `recommended`, I intends to recommend specifying the check names instead of `NOLINT` alone.